includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -86,9 +87,47 @@ function get_columns() {
 		// Backwards compatible filter name, moving forward should have single filter name
 		$columns = apply_filters( 'sensei_analysis_overview_' . $this->type . '_columns', $columns, $this );
 		$columns = apply_filters( 'sensei_analysis_overview_columns', $columns, $this );
+		// We want to fetch the totals only once.
+		remove_filter( 'sensei_analysis_overview_columns', array( $this, 'add_totals_to_report_column_headers' ) );
```

Comment:
Hey Imran, I don't think you would need to remove the filter once #4904 is merged. Donna solved the multiple calls issue there in https://github.com/Automattic/sensei/pull/4904/commits/9cefaf7d62294f59fcf1a038d7f9dd45f9e52516#diff-6ce1dc075c4832d5bf050e621bcfecbab91d97504e8bc565f4f476b1ea2f654dR56-R103. Basically the columns get cached after the first call. You could copy the solution and resolve the conflicts later or wait for her PR to get merged.